packages/components/src/components/tag-dismissible/tag-dismissible.tsx

```diff
@@ -37,14 +42,16 @@ export class TagDismissible {
 
   public render(): JSX.Element {
     validateProps(this, propTypes);
-    attachComponentCss(this.host, getComponentCss, this.color, !!this.label);
+    attachComponentCss(this.host, getComponentCss, this.color, !!this.label, this.theme);
 
     const PrefixedTagNames = getPrefixedTagNames(this.host);
     return (
       <button type="button" {...parseAndGetAriaAttributes(this.aria)}>
         <span class="sr-only">Remove:</span>
-        {this.label && <span class="label">{this.label}</span>}
-        <slot />
+        <span>
```

Comment:
Extra span?

Reply:
Needed to ensure the label and slotted text  are not next to each other. Also if `<i>`, `<bold>` etc. is used in the slot, it would get the flex gap.